lib/modules/platform/bitbucket/schema.ts

```diff
@@ -74,3 +74,24 @@ export const Repositories = z
     values: LooseArray(RepoInfo),
   })
   .transform((body) => body.values);
+
+const taskState = z.union([z.literal('RESOLVED'), z.literal('UNRESOLVED')]);
```

Comment:
```suggestion
const TaskState = z.union([z.literal('RESOLVED'), z.literal('UNRESOLVED')]);
```
we use an starting uppercase letter for schemas. typescript can handle this